packages/prop-types/src/utils/index.js

```diff
@@ -39,3 +39,18 @@ export const getDisplayName = Component => {
 }
 
 export const isNil = value => value == null
+
+// NOTE: example from MDN docs
+//       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value#Examples
+export const getCircularReplacer = () => {
+  const seen = new WeakSet()
```

Comment:
Verified that babel-built version of a package (ie, ActionMenu) falls back to Set if WeakSet not an available function.

Also found out that the element-of-type.js code is *included* in a webpack-bundled app in production mode.  This isn't great.  I would have thought the propTypes info would have been cleared right out.  This is not fault of this PR, but I think we should be stripping that bloat out in prod.